package.json

```diff
@@ -23,13 +23,14 @@
     "@mapbox/mapbox-gl-language": "^0.10.1",
     "@yext/answers-core": "^1.1.0-beta.0",
     "@yext/answers-storage": "^1.0.0-beta.0",
-    "@yext/rtf-converter": "^1.2.0",
+    "@yext/rtf-converter": "^1.3.0",
     "cross-fetch": "^3.0.6",
-    "css-vars-ponyfill": "^2.3.1",
+    "css-vars-ponyfill": "^2.4.3",
     "gulp-sourcemaps": "^2.6.5",
     "handlebars": "^4.7.2",
     "js-levenshtein": "^1.1.6",
     "kind-of": "^6.0.3",
+    "lodash": "^4.17.21",
```

Comment:
it looks like we are adding `lodash` for the first time here, is that intentional?